Debug/DataCollector.php

```diff
@@ -51,11 +54,21 @@ public function reset(): void
         $this->data['metadata_files'] = [];
         $this->data['loaded_dirs'] = [];
         $this->data['runs'] = [];
+        $this->data['instance'] = $this->instance;
+    }
+
+    public function getInstance(): string
+    {
+        return $this->data['instance'];
     }
 
     public function getName(): string
     {
-        return 'jms_serializer';
+        if (($this->instance ?? $this->data['instance']) === 'default'){
```

Comment:
Looking at constructor - `$this->instance` is not nullable so `??` seems to be redundant.

Reply:
yea, this is something that I still need to figure out. there is something different when the data collector is created and used in the profiler versus when is used to collect the data while running the application.

i will look better into this